openapi2beans/pkg/generator/JavaStructures.go

```diff
@@ -0,0 +1,72 @@
+package generator
+
+type JavaPackage struct {
+	Name            string
+	Classes         map[string]*JavaClass
+	Enums           map[string]*JavaEnum
+}
+
+func NewJavaPackage(name string) *JavaPackage {
+	javaPackage := JavaPackage{
+		Name:            name,
+		Classes:         make(map[string]*JavaClass),
+		Enums:           make(map[string]*JavaEnum),
+	}
+	return &javaPackage
+}
+
+type JavaClass struct {
+	Name                string
+	Description         []string
+	JavaPackage         *JavaPackage
+	DataMembers         []*DataMember
```

Comment:
Is this list supposed to be sorted ?
Maybe a comment ?

Reply:
hmm, there is a function that sorts these, but it's not in this file, do you think it should be moved?

Reply:
I've added a function to JavaClass to sort each of these and moved the sorting from within the code to there.